### PR TITLE
Feature implement mailer

### PR DESCRIPTION
**Pull request by KarenVentura**
@anymoto @FerPerales

### CODE REVIEW

File: app/helpers/post_location_helper.rb [hunk 1]
```diff
@@ -8,7 +8,6 @@ def link_to_post(post)
   end
 
   def show_votes(post)
-    if post.internal_post?
       if post.has_voted?(current_user)
```
Comment (FerPerales) [inline app/helpers/post_location_helper.rb]: Check indentation

File: app/models/external_post.rb [hunk 2]
```diff
@@ -1,5 +1,6 @@
 class ExternalPost < Post
   validates :external_id, uniqueness: true
+  after_create :create_question_tag
```
Comment (FerPerales) [inline app/models/external_post.rb]: Should we move this to `Post`?

**Comment by anymoto** [comment]
LGTM :+1:

**Comment by FerPerales** [comment]
Besides comments, LGTM

**Pull request merged**